src/main/scala/firrtl/passes/RemoveValidIf.scala

```diff
@@ -0,0 +1,26 @@
+package firrtl
+package passes
+import firrtl.Mappers.{ExpMap, StmtMap}
+
+// Removes ValidIf as an optimization
+object RemoveValidIf extends Pass {
```

Comment:
Legalize was intended to be a pass for miscellaneous things that are required for Verilog Legalization. Perhaps this pass should be merged with that?